DomainServices/CustomerBankInfos/CustomerBankInfoService.cs

```diff
@@ -0,0 +1,102 @@
+﻿using DomainModels;
+using DomainServices.CustomerBankInfos.Interface;
+using EntityFrameworkCore.UnitOfWork.Interfaces;
+using Infrastructure.Data.Context;
+
+namespace DomainServices.CustomerBankInfos
+{
+    public class CustomerBankInfoService : ICustomerBankInfoService
+    {
+        private readonly IUnitOfWork _unitOfWork;
+        private readonly IRepositoryFactory _repositoryFactory;
+
+        public CustomerBankInfoService(IUnitOfWork<FeatureContext> unitOfWork,
+            IRepositoryFactory<FeatureContext> repositoryFactory)
+        {
+            _unitOfWork = unitOfWork ?? throw new ArgumentNullException(nameof(unitOfWork));
+            _repositoryFactory = repositoryFactory ?? (IRepositoryFactory)_unitOfWork;
+        }
+
+        public void Deposit(long customerId, decimal amount)
+        {
+            var _customerBankInfoRepo = _unitOfWork.Repository<CustomerBankInfo>();
+
+            var query = _customerBankInfoRepo.SingleResultQuery().AndFilter(x => x.CustomerId.Equals(customerId));
+
+            var bankInfo = _customerBankInfoRepo.FirstOrDefault(query);
+
+            if (bankInfo == null)
+            {
+                throw new ArgumentNullException($"User Not Found with this Id: {customerId}");
+            }
```

Comment:
User not found? está certo isso?
```suggestion
            var bankInfo = _customerBankInfoRepo.FirstOrDefault(query)
                ?? throw new ArgumentNullException($"User Not Found with this Id: {customerId}");
```